doc/man1/ciphers.pod

```diff
@@ -156,19 +156,20 @@ The cipher string B<@SECLEVEL=n> can be used at any point to set the security
 level to B<n>, which should be a number between zero and five, inclusive.
 See L<SSL_CTX_set_security_level> for a description of what each level means.
 
+The cipher list can be prefixed with B<DEFAULT> keyword, which enables
+the default cipher list as defined below.  Unlike the cipher string,
+this prefix may not be combined with other strings using B<+> character.
+For example, B<DEFAULT+DES> is not a valid cipher list.
```

Comment:
s/valid cipher list/valid cipher string/

Reply:
Mind you, the real story is more subtle (back to the BUGS issue), since `DEFAULT+FOO` is surprisingly instead "DEFAULT:+FOO", which would put FOO last (for any value of FOO). So the users who specify "DEFAULT+FOO" do not get anything like what they probably intended.  If that's described, then it goes into a BUGS section, and must not be described as a stable interface.

Reply:
Fixed the *the*.  See below for comment on BUGS.